app/serializers/disability_claim_base_serializer.rb

```diff
@@ -17,31 +17,31 @@ def date_filed
   end
 
   def min_est_date
-    date_from_string 'minEstClaimDate'
+    date_from_string 'min_est_claim_date'
```

Comment:
One idea, create a thing like `dates_from_strings :min_est_claim_date, :max_est_claim_date` etc. that converts these en mass.